Sources/GRPC/LengthPrefixedMessageReader.swift

```diff
@@ -31,13 +31,22 @@ import Logging
 /// - SeeAlso:
 /// [gRPC Protocol](https://github.com/grpc/grpc/blob/master/doc/PROTOCOL-HTTP2.md)
 internal struct LengthPrefixedMessageReader {
-  /// The mechanism that messages will be compressed with.
   var compression: CompressionAlgorithm?
+  var decompressor: Zlib.Inflate?
```

Comment:
Seems a bit hard-coded right now, but we can still refactor this later if we ever need to.

Reply:
Agreed. I'm okay with it for now, if we add compression not from zlib then we'll definitely need to change this.